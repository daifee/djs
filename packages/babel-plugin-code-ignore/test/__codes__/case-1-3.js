

import error from './error';// @code-ignore-statement kuwo
import success from './success';

