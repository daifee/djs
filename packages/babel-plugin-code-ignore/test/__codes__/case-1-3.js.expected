"use strict";

var _success = _interopRequireDefault(require("./success"));
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }